From 86aa849f838b746d3978289cf429048924ee0931 Mon Sep 17 00:00:00 2001
From: Tatsuro BABA <t.baba@grooves.co.jp>
Date: Tue, 6 Jul 2010 11:56:12 +0900
Subject: [PATCH] Failing test for nested resources unexpectedly inheriting :except

:except option on parent resource should not be inherited to its
children.
see #5048
---
 actionpack/test/dispatch/routing_test.rb |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/actionpack/test/dispatch/routing_test.rb b/actionpack/test/dispatch/routing_test.rb
index 463a62c..8611276 100644
--- a/actionpack/test/dispatch/routing_test.rb
+++ b/actionpack/test/dispatch/routing_test.rb
@@ -882,6 +882,10 @@ class TestRoutingMapper < ActionDispatch::IntegrationTest
       assert_equal 'comments#index', @response.body
       assert_equal '/posts/1/comments', post_comments_path(:post_id => 1)
 
+      get '/posts/1/comments/1/edit'
+      assert_equal 'comments#edit', @response.body
+      assert_equal '/posts/1/comments/edit', edit_post_comments_path(:post_id => 1)
+
       post '/posts'
       assert_equal 'pass', @response.headers['X-Cascade']
       put '/posts/1'
-- 
1.7.1

